aws-blog-jobserver-emr/html/mltest.html

```diff
@@ -0,0 +1,284 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
```

Comment:
Agreed - but for this example I'd like to keep it in a single file as it is not the primary focus of the blogpost / codebase.